parlai/crowdsourcing/tasks/turn_annotations_static/webapp/src/components/task_components.jsx

```diff
@@ -120,6 +120,7 @@ var handleSubtaskSubmit = function (subtaskIndex, numSubtasks, initialTaskData,
     mephistoSubmit(window.workerAnswers);
   }
   showDisableCssNextButton();
+  setIndex(subtaskIndex + 1); 
```

Comment:
moving setIndex from old L233 to here (in old version, index (subtask index get bumped to next +1 regardless of whether the turker input is valid or not)